docs/content/middlewares/http/compress.md

```diff
@@ -255,3 +255,185 @@ http:
   [http.middlewares.test-compress.compress]
     defaultEncoding = "gzip"
 ```
+
+### `defaultEncoding`
```

Comment:
This documentation section seems to be duplicated.

Reply:
Oops.. How could that happen :-|